File: easy/TwoSum.js
let nums = [5,5]
let target = 10

var twoSum = function(nums, target) {
    let numsObj = {}
    for (let i = 0; i < nums.length; ++i ){
        if (numsObj[nums[i]] !== undefined && i !==numsObj[nums[i]])
            {
                console.log([i, numsObj[nums[i]]])
                return[i, numsObj[nums[i]]]
            }        
        numsObj[target-nums[i]] = i
    }
};

twoSum(nums, target)

File: C#/Leetcode/Leetcode/easy/twoSum.cs
namespace Leetcode.easy
{
    internal class twoSum
    {

        public int[] TwoSum(int[] nums, int target)
        {
            Dictionary<int, int> dict = new Dictionary<int, int>();
            for (int i = 0; i < nums.Length; i++)
            {
                if (dict.ContainsKey(nums[i]))
                {
                    return [dict[nums[i]], i];
                }
                dict[target - nums[i]] = i;
            }
            return [];
        }
    }
}
